.gitlab/validate-agent-build/trigger_agent_build.py

```diff
@@ -4,15 +4,20 @@
 
 
 DATADOG_AGENT_PIPELINE_URL = os.environ['DATADOG_AGENT_PIPELINE_URL'].rstrip('/')
+VERSION_TAG = os.environ['VERSION_TAG']
 BASE_URL = os.environ['CI_API_V4_URL']
 CI_TOKEN = os.environ['CI_JOB_TOKEN']
 
 
 def trigger_pipeline():
+    trigger_ref = VERSION_TAG
+    if not VERSION_TAG.startswith("7."):
```

Comment:
should we check if `VERSION_TAG` exists instead of checking if it starts with 7?

Reply:
Now checking the datadog-agent branch https://github.com/DataDog/integrations-core/pull/10842/files#diff-b4d35ba9894a41c951ead81b42de092804a0bff4c23d60660c00a6cbddd639e6R16